src/index.html

```diff
@@ -15,8 +15,102 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo-container">
+        <a>
+          <img
+            src="images/logo.png"
+            alt="Company Name Logo"
+          />
+        </a>
+      </div>
+
+      <nav>
+        <ul class="nav-list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
```

Comment:
Don't use empty `href` attributes. It's better to use a more descriptive value, such as the id of the section this link should lead to. For example, `href="#samsung"`.